Framework/Algorithms/src/GetEiMonDet2.cpp

```diff
@@ -38,11 +39,11 @@ const static std::string FIT_STATUS_SUCCESS("success");
  */
 namespace IndexTypes {
 /// Tag for detector ids
-const static std::string DETECTOR_ID("DetectorID");
+const static std::string DETECTOR_ID("Detector ID");
```

Comment:
Is there a danger of breaking user scripts here that would have used `DetectorID` with no space? From the usage stats I see that this has only ever been used as a child algorithm but I can't find the usage anywhere in the code base.